docs/guide/writing-tasks.md

```diff
@@ -422,10 +422,11 @@ And some environment variables can be set to control behavior of the Cirrus CI A
 Name | Default Value | Description
 ---  | --- | ---
 CIRRUS_CLONE_DEPTH | `0` which will reflect in a full clone of a single branch | Clone depth.
+CIRRUS_CLONE_DIR | Defaults to `$CIRRUS_WORKING_DIR` if not empty and doesn't contain `$CIRRUS_CLONE_DIR` itself. Othervise `CIRRUS_CLONE_DIR` is set to `cirrus-ci-build` folder inside of a system's temporary folder | Directory to put repository sources into.
```

Comment:
I think this needs new definition to be introduced across the documentation (e.g. a _scratch directory_) to simplify the explanation.

Otherwise the reader needs to keep the track of all these cross-references between `CIRRUS_CLONE_DIR` and `CIRRUS_WORKING_DIR` in their memory.